.circleci/config.yml

```diff
@@ -58,13 +57,27 @@ jobs:
             export PATH="$(yarn bin):$PATH"
             pipenv run make html
 
+  cover:
+    <<: *defaults
+    steps:
+      - checkout
+      - <<: *install
+      - run: npm run cover
+      - run: npm run coveralls
+
 workflows:
   version: 2
   test-publish:
     jobs:
       - test-node8
       - test-node10
       - test-node12
-      - test-smoke
+      - test-smoke:
+          requires:
+            - test-node12
```

Comment:
IMO, I wouldn't depend test-node12, this delays running smoke tests (which are a little slow as it builds Protagonist) by 30 more seconds. Earlier we start running test-smoke the sooner it finishes.